src/test/java/bio/terra/service/snapshot/flight/create/AddCreatedSnapshotIdToSnapshotRequestStepTest.java

```diff
@@ -0,0 +1,48 @@
+package bio.terra.service.snapshot.flight.create;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+import bio.terra.common.category.Unit;
+import bio.terra.service.snapshotbuilder.SnapshotRequestDao;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.StepResult;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+@Tag(Unit.TAG)
+class AddCreatedSnapshotIdToSnapshotRequestStepTest {
+  @Mock private SnapshotRequestDao snapshotRequestDao;
+  private static final UUID SNAPSHOT_REQUEST_ID = UUID.randomUUID();
+  private static final UUID CREATED_SNAPSHOT_ID = UUID.randomUUID();
+  private AddCreatedSnapshotIdToSnapshotRequestStep addCreatedSnapshotIdToSnapshotRequestStep;
+  private final FlightContext flightContext = mock(FlightContext.class);
```

Comment:
I don't think it's needed for this test, but if you did want to keep it, I would use `@Mock` for this field too.